qa/workunits/cephadm/test_cephadm.sh

```diff
@@ -366,8 +366,8 @@ $CEPHADM unit --fsid $FSID --name mon.a -- is-enabled
 ## shell
 $CEPHADM shell --fsid $FSID -- true
 $CEPHADM shell --fsid $FSID -- test -d /var/log/ceph
-expect_false $CEPHADM --timeout 1 shell --fsid $FSID -- sleep 10
-$CEPHADM --timeout 10 shell --fsid $FSID -- sleep 1
+expect_false $CEPHADM --timeout 10 shell --fsid $FSID -- sleep 60
+$CEPHADM --timeout 60 shell --fsid $FSID -- sleep 10
```

Comment:
waiting for 10s just for triggering a timeout is excessive. what about moving this to a unit test instead?

Reply:
Seems minor compared to the duration of many of the tests in this script. How would moving this to a unit help?